### PR TITLE
Setting up linters

### PR DESCRIPTION
**Pull request by Del-sama**
**What does this PR do?**

- It adds a screenshot of ESlint Airbnb style violation

- It adds a screenshot of HoundCI commenting on a PR

- It adds screenshots of code coverage and GPA badges in my READMe file
- It adds  a screencast showing gulp watching and refreshing the browser when changes are made to HTML, CSS & JS files

**Description of Task to be completed?**

- Deliver two screenshots showing Eslint Airbnb style violations and HoundCI commenting on a PR

- Deliver a screenshot of your code coverage and GPA badges in your repo's README file
- Deliver a screencast showing gulp watching and refreshing your browser when changes are made to your HTML, CSS & JS files

**How should this be manually tested?**

- By looking at the screenshots in the output folder of "Setting-up-linters"

-  By looking at the screenshots in the output folder of "Checking repository health"
-  By clicking on the youtube click in the output folder of "Automating tasks" 

**Any background context you want to provide?**

Targeted learning outcomes include:

- End-to-End testing

- Test Driven Development

- Attention to detail

- Command line

- Git

- Version Control

- Debugging
- HTML & CSS

**What are the relevant pivotal tracker stories?**

N/A
**Screenshots (if appropriate)**
eslint airbnd style violation

houndci comment on a pr
code coverage and gpa badges

**Questions:**

N/A

### CODE REVIEW

**Comment by raphaelakpan** [comment]
Looks good. 

But from the look of things, you have set up `eslint` to lint only one file `lintertest.js` which is okay for testing that your `eslint` works. However, you would want to rather test your folder that contains your source files.  So when you start creating your source files, your command should lint the entire folder instead of linting a single file.

**Comment by CodeMuhammed** [comment]
The houdci and linter setup looks good.
Just make sure that codeclimate GPA and coverage status    displays their real values in your readme instead of unknowns.

File: 06-building-a-program-start-to-finish/setting-up-your-repository/Output/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+https://github.com/Andela-ddiei/Inverted-Index
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/setting-up-your-repository/Output/README.md]: Kindly use the proper syntax for URLs in Markdown.

File: 06-building-a-program-start-to-finish/automating-tasks/Output/README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#### Output 
+- Link to youtube screencast 
+[youtube.com/Delores-Diei](https://www.youtube.com/watch?v=2qB_ryZe56c)
```
Comment (michael-alade) [inline 06-building-a-program-start-to-finish/automating-tasks/Output/README.md]: The videos shows what I would want for hot reloading. Looks Good :+1:

Comment (raphaelakpan) [reply 06-building-a-program-start-to-finish/automating-tasks/Output/README.md]: Look good! The browser hot reloading works fine.

Comment (q-ode) [reply 06-building-a-program-start-to-finish/automating-tasks/Output/README.md]: I only see reloading of HTML files.

None for CSS/JS files.

File: 06-building-a-program-start-to-finish/setting-up-your-repository/Output/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+#### OUTPUT
+
+
+- Link to Github repo 
+  
+  
+[andela-diei/inverted-index](https://github.com/andela-ddiei/inverted-index.git)
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/setting-up-your-repository/Output/README.md:7]: You `ReadMe` is much better now.

**Review by q-ode — COMMENTED** [review]
Apart from Travis all your other badges are not displayed yet. How can we fix this?

File: 06-building-a-program-start-to-finish/setting-up-your-repository/Output/README.md (hunk 3 above)
Comment (Del-sama) [reply 06-building-a-program-start-to-finish/setting-up-your-repository/Output/README.md:7]: I haven't written any tests yet. I'll do that and push.

File: 06-building-a-program-start-to-finish/automating-tasks/Output/README.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+#### Output 
+- Link to youtube screencast 
+[youtube.com/Delores-Diei](https://youtu.be/9e2fKH2s8CQ)
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/automating-tasks/Output/README.md:3]: I can't access this video.

File: 06-building-a-program-start-to-finish/setting-up-your-repository/Output/README.md (hunk 3 above)
Comment (q-ode) [reply 06-building-a-program-start-to-finish/setting-up-your-repository/Output/README.md:7]: Ok.

I'll expect this to change during the course of your project execution.

**Pull request closed without merging**